dev-packages/cli/package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "@theia/application-manager": "1.23.0",
     "@theia/application-package": "1.23.0",
+    "@theia/core": "1.23.0",
```

Comment:
I'm not happy with this dependency in our `cli`. It shouldn't really influence anything, since any project which uses `@theia/cli` probably also has `@theia/core` in there, but its still quite nasty.

We need this currently to use the `BackendRequestService` in the `ovsx` and `cli` package, but I could refactor this into its own `dev-package`. WDYT?

Reply:
> I'm not happy with this dependency in our cli

@msujew I agree, if possible I would like to avoid coupling `core` into `cli`. The refactor you mentioned seems good to me.

Reply:
Thanks for the Input Vince, I've added `@theia/request-service` which is used by `@theia/ovsx-client`, `@theia/cli` and `@theia/core`.